src/filtering.jl

```diff
@@ -104,6 +112,32 @@ end
 # Macro used by RAI (corporate sponsor of this package)
 # TODO: drop support for this when RAI codebase is fully switched to ReTestItems.jl
 const ___RAI_MACRO_NAME_DONT_USE = Symbol("@test_rel")
+function __filter_rai(f, expr)
```

Comment:
FYI this duplicates a lot of code from `try_get_tags` but i just really want this code to be trivial to delete (when the day comes) so i'd rather repeat some very similar logic than tie it into a function that we're probably going to keep